tests/scripts/installandtest.sh

```diff
@@ -18,14 +24,15 @@ if [ -z "${SKIP_INSTALL}" ]; then
 fi
 $HOME/peak/run.sh ${TESTS_REGEX}
 
+if [ -z "${SKIP_PODS_OUTPUT}" ]; then
```

Comment:
@crobby I moved the pod and operator logs out of the test error check so that we always save these logs as artifacts during a test run. Can I remove `SKIP_PODS_OUTPUT` as an variable since it only appears to be used to toggle to the test log?

Reply:
Yup, I added that to be able to prevent the pod output print when running on a local cluster

Reply:
Update. I removed `SKIP_PODS_OUTPUT`

Reply:
This sounds good to me.  Sorry I'm way late to the party